src/components/overview/OverviewMain.jsx

```diff
@@ -140,12 +140,8 @@ class overviewMain extends React.Component {
     }
   }
 
-  componentDidMount() {
-    //when the component mounts, this gets the API data, sets it as the store, and
-    //then organizes it and saves it as the state
-    let product_id = 3; //filler
-    //componentDidMount, grab the API data for the product info
-    Axios.get(`http://3.134.102.30/products/${product_id}`)
+  getData(page_id) {
```

Comment:
Possibly look into pulling these two functions out of this file and putting them into their own file to separate concerns.